state/migration_export.go

```diff
@@ -1684,7 +1691,7 @@ func (e *exporter) checkUnexportedValues() error {
 	}
 
 	for key := range e.statusHistory {
-		if !e.cfg.SkipInstanceData && !strings.HasSuffix(key, "#instance") {
+		if !e.cfg.SkipInstanceData && (!strings.HasSuffix(key, "#instance") && !strings.HasSuffix(key, "#modification")) {
```

Comment:
I feel like this should be 
!(instance || modification)

Reply:
Yeah, that is more readable.